### PR TITLE
Warden whitelist

### PR DESCRIPTION
**Pull request by TadJohnson00**
Make the warden whitelisted, increase playtime requirements. No more redshirts with RPGs, show a little class.

## About the PR

warden now requires whitelist
## Why / Balance

no more shitwarden, no more wardenloose
## Technical details

technically wild
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Warden requirements now include mandatory officer time and whitelist.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
take my reddit gold, very nice

**Review by DebugOk — DISMISSED** [review]
Me omw to sign off on this:tm:

File: Resources/Prototypes/Roles/Jobs/Security/warden.yml [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
-#      role: JobSecurityOfficer
-#      time: 36000 #10 hrs
-    - !type:DepartmentTimeRequirement # DeltaV - Security time requirement
-      department: Security
-      time: 36000 # 10 hours
+    - !type:RoleTimeRequirement # DeltaV - JobSecurityOfficer time requirement. Make them experienced in proper officer work.
+      role: JobSecurityOfficer
+      time: 43200 # DeltaV - 12 hrs
+    - !type:DepartmentTimeRequirement # DeltaV - JobDetective time requirement. Give them an understanding of basic forensics.
+      department: JobDetective
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Security/warden.yml]: department: JobDetective 💀

Comment (DangerRevolution) [reply Resources/Prototypes/Roles/Jobs/Security/warden.yml]: `role: JobDetective`

**Review by DebugOk — APPROVED** [review]
Approval part 2

**Comment by deltanedas** [comment]
take my reddit silver kind sir

**Pull request merged**